playbooks/openstack-osh_hostconfig.yml

```diff
@@ -28,10 +30,12 @@
         value: "false"
 
     - name: Workaround stupid JeOS image bugs (bsc#1138727)
```

Comment:
BTW, wasn't this problem already fixed in latest JeOS image?

Reply:
Yes, but it seems that it wasnt correctly fixed, new fix is from https://github.com/SUSE-Cloud/socok8s/pull/600